index.test.ts

```diff
@@ -131,6 +132,8 @@ afterEach(() => {
 // Redis is required to handle staging(?) of S3 files to be pushed to snowflake
 let redis: Redis | undefined;
 let cache: any
+let postgres: Pool | undefined;
+let storage: any
 
 beforeAll(() => {
     redis = new Redis()
```

Comment:
we only depend on Redis for `set` and `get` now. Could get rid of the dependency and mock as well?

Reply:
I kind of want to actually pull the redis and postgres out into a plugin dev starting kit.